### PR TITLE
add functions for scaling home and work network multiplers per network

### PR DESCRIPTION
**Pull request by brynmathias**

### CODE REVIEW

File: src/constant.c [hunk 1]
```diff
@@ -31,10 +31,6 @@ const int AGE_TYPE_MAP[N_AGE_GROUPS] = {
 	AGE_TYPE_CHILD,
 	AGE_TYPE_CHILD,
 	AGE_TYPE_ADULT,
-	AGE_TYPE_ADULT,
-	AGE_TYPE_ADULT,
```
Comment (roberthinch) [inline src/constant.c:35]: Removing these 4 AGE_TYPE_ADULT here is causing the failing test. 
AGE_TYPE_MAP is used to for assigning the age_type which is then used by the individual to see which parameters to use for calculating the number of random interactions.

Comment (brynmathias) [reply src/constant.c]: This fixes my tests, thanks!

**Comment by brynmathias** [comment]
> In tests/constant.py, we still have the constant WORK, whereas in the C code it is has now all been changed to OCCUPATION. Please can you update the Python test code to make sure that the constant names are in line throughout, thanks (it is used in 3 or 4 tests).

Done

File: src/params_utils.i [hunk 2]
```diff
@@ -1383,12 +1384,13 @@ int set_param_daily_non_cov_symptoms_rate(parameters *params, double value)
...
 /*****************************************************************************************
-*  Name:		set_param_lockdown_work_network_multiplier
+*  Name:		set_param_lockdown_occupation_multiplier
 *  Description: Sets the value of parameter
 ******************************************************************************************/
-int set_param_lockdown_work_network_multiplier(parameters *params, double value)
+int set_param_lockdown_occupation_multiplier( parameters *params, double value, int idx)
 {
-    params->lockdown_work_network_multiplier = value;
+    if (idx >= N_OCCUPATION_NETWORKS) return FALSE;
```
[the remark this answers is not in the document]
Comment (brynmathias) [reply src/params_utils.i]: Created an ERROR type def and return this thoughout in the params setting / getting now when there's an error

**Pull request merged**